db/index.js

```diff
@@ -0,0 +1,92 @@
+const db = require('mongoose');
+require('mongoose-currency').loadType(db);
+
+const { Currency } = db.Types;
+
+db.connect('mongodb://localhost:27017/ontarget', { useNewUrlParser: true });
+
+db.connection.on('connected', () => {
+  // eslint-disable-next-line no-console
+  console.log('connected to MongoDB');
+});
+
+db.connection.on('error', (err) => {
+  // eslint-disable-next-line no-console
+  console.log('Error occurred: ', err);
+});
+
+/*
+  Schema Definition
+*/
+const reviewSchema = db.Schema({
```

Comment:
i know you mentioned this in standup but i learned that its considered a best practice to put your models in separate files